include/VecCore/Backend/Implementation.h

```diff
@@ -284,6 +285,21 @@ VECCORE_FORCE_INLINE VECCORE_ATT_HOST_DEVICE constexpr Bool_s EarlyReturnAllowed
 #endif
 }
 
+// An extension of EarlyReturnAllowed(), and new name is a reminder that it can be used for other purposes as well
+template <typename T, size_t simdSize = 1>
+VECCORE_FORCE_INLINE
+VECCORE_ATT_HOST_DEVICE
+constexpr Bool_s SIMDsizeUpTo()
```

Comment:
No, but it's already possible to do that with VecCore. What you want is a slightly shorter version of it wrapped inside a function, and I believe that this part is specific to VecGeom.